static.config.js

```diff
@@ -149,6 +150,7 @@ const getTopicPageData = async (id, parent_topic_collection, client) => {
   )
     .filter(page => !topLinkIds.includes(page.id))
     .map(page => ({
+      __typename: page.__typename,
```

Comment:
we could probably come up with a name we like better and feels less internal/debuggy than this

Reply:
pageName

Reply:
Except we've learned that isn't really the page/content type but the name of the graphql node